rice/flags.go

```diff
@@ -12,6 +12,7 @@ import (
 var flags struct {
 	Verbose     bool     `long:"verbose" short:"v" description:"Show verbose debug information"`
 	ImportPaths []string `long:"import-path" short:"i" description:"Import path(s) to use. Using PWD when left empty. Specify multiple times for more import paths to append"`
+	BuildTags []string	`long:"tags" short:"t" description:"list of build tags to consider satisfied during the embedding."`
```

Comment:
I think it would make sense to have a bit more specific flag for this. How about `--go-build-tags` ?
This makes it immediately clear to any reader what kind of tags are specified here.